aws/resource_aws_guardduty_publishing_destination_test.go

```diff
@@ -0,0 +1,292 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/guardduty"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("aws_guardduty_publishing_destination", &resource.Sweeper{
+		Name: "aws_guardduty_publishing_destination",
```

Comment:
In the `aws_guardduty_detector` sweeper, we should add `Dependencies: []string{"aws_guardduty_publishing_destination"},`